.github/workflows/pr.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.15.5'
+          go-version: '1.16beta1'
```

Comment:
please use the update go version in the hack folder
to update it everywhere in master.yml and also the jenkins scripts